### DISCUSSION TITLE
Recommend Bonus Claims Limit

### DISCUSSION THREAD

**Discussion by KennethHF** [GriefPrevention/GriefPrevention#284] [Ideas]
### What behaviour is observed:
No issue with the plugin; recommending an additional feature.
### What behaviour is expected:
Would like to see a max limit enforced for the bonus claim limit.
### Steps/models to reproduce:
I found in the source code where this could be implemented (see highlighted text in screenshot below).  Of course, this screenshot is hard coded.  I would recommend a variable be used in the config file.

[Screenshot](http://www.mediafire.com/view/0o56sd3tfsp1ga8/codeGP.png)

### Spigot/Craftbukkit/Paper and GriefPrevention version:
Version 16.8

### Stack trace/error

```
N/A
```

### GriefPrevention config.yml (if applicable)

```N/A
```

**Comment by bigpresh** [comment]
My initial feeling on this is that whatever is adding bonus blocks should be responsible for not adding more than they should have.

However, I suppose that adds complexity to whatever does it (e.g. an adjustclaimblocks command in a Votifier setup, for instance, is trivial, but having it first pull their current bonus blocks amount and check it against a threshold suddenly gets more complicated).
